### DISCUSSION TITLE
다음 메인 브랜치 머지 일정?

### DISCUSSION THREAD

**Discussion by pekochan069** [yiminwook/liveuta-asis#107] [General]
@yiminwook 
난 이브나 크리스마스 당일에 하면 괜찮을거같음
큼직한건 대충 다 한거같은데

**Comment by yiminwook** [comment]
자잘한거 좀 수정할거 있을 것 같긴한데 주말에 시간이 충분하지 않을 것 같아서 일단 12월 안에?

**Comment by pekochan069** [comment]
ㅇㅋ

**Reply by yiminwook** [reply]
일단 당장 눈에 보이는건 다 한것 같은데 배포해서 운영 돌려볼까?

**Reply by pekochan069** [reply]
세트리스트에 채널 아이콘 추가하고 있으니까 한 10분만

**Reply by pekochan069** [reply]
일단 할만한건 다 한듯

**Comment by pekochan069** [comment]
갤에 올리려고 적은 체인지로그인데 뭐 빠트린거 있나?

1. 홈 페이지 추가
1.1. 버우갤 대문 이미지
1.2. 스케줄
1.3. 최근에 추가된 채널

2. 사이트 전체 스타일 리마스터
2.1. 상단바, 모바일 하단바
2.2. 모바일 사이드바가 모바일 하단바에 이동
2.3. 팝업이 필요한 곳에 아주 좋은거 사용함(하단바 더보기, 세트리, 세트리 작성 폼 등)
2.4. 스케줄 페이지 들어갈 때 빙글빙글 로딩 대신 스켈레톤 사용
2.5. 스케줄 카드 기능들(샤로운 탭에서 열기, 알림설정 등) 위치 이동
2.6. 세트리 페이지
2.7. 채널 페이지에서 브라우자 크기가 일정 크기 이상일시 그리드로 변경
2.8. 다크 모드 지원 강화
2.9. 전반적으로 좀 더 깔끔하게 만드려고 함
2.10. 그 외 자잘한 스타일 변경점들이 좀 많음

3. 스케줄 성능 향상

4. 전반적인 성능 향상(아마? 내부적으로 변경된게 믾아서 성능이 좋아질거라고 생각함. 아님 말고)

5. 설정 추가 - 스케줄 자동 동기화 온오프

**Reply by yiminwook** [reply]
1.1. 버우갤 대문 이미지 이건 바뀔 수 잇어서.. 대문만 올릴건 아님

지금 채널페이지 최근 등록순으로 정렬 SegmentConrol 만드는 중 잠만

**Reply by pekochan069** [reply]
1. ​**홈 페이지 추가​** 
- 1.1. 대문 이미지(버우갤 대문 등을 사용할 예정) 
- 1.2. 스케줄 
- 1.3. 최근에 추가된 채널 

2. ​**사이트 전체 스타일 리마스터​** 
- 2.1. 상단바, 모바일 하단바 
- 2.2. 모바일 사이드바가 모바일 하단바에 이동 
- 2.3. 팝업이 필요한 곳에 아주 좋은거 사용함(하단바 더보기, 세트리, 세트리 작성 폼 등) 
- 2.4. 스케줄 페이지 들어갈 때 빙글빙글 로딩 대신 스켈레톤 사용 
- 2.5. 스케줄 카드 기능들(로운 탭에서 열기, 알림설정 등) 위치 이동 
- 2.6. 세트리 페이지 
- 2.7. 채널 페이지에서 브라우저 크기가 일정 크기 이상일시 그리드로 변경 
- 2.8. 다크 모드 지원 강화 - 채널 등록 페이지 제외하면 전부 다크모드 제대로 지원 
- 2.9. 전반적으로 좀 더 깔끔하게 만드려고 함 
- 2.10. 그 외 자잘한 스타일 변경점들이 좀 많음 

3. ​**스케줄 성능 향상​**

4. ​**전반적인 성능 향상**​(아마? 내부적으로 변경된게 믾아서 성능이 좋아질거라고 생각함. 아님 말고) 

5. ​**설정 추가​**
​- ​5.1. 스케줄 자동 동기화 온오프



앞으로 할 예정인 것들
1. 채널, 세트리 기능 강화
2. 백엔드 최적화로 더 빠른 속도

**Reply by yiminwook** [reply]
세트리 강화 할 수 있으려나?

**Reply by pekochan069** [reply]
몰루 그냥 써봄

**Reply by pekochan069** [reply]
아니면 우타툴즈에 있는 세트리 작성 기능을 여기에 가져와도 되고

**Reply by yiminwook** [reply]
다했음 이제 배포할게

**Reply by pekochan069** [reply]
ㅇㅋ
